samza-rest/src/main/java/org/apache/samza/rest/model/ApplicationInfo.java

```diff
@@ -0,0 +1,294 @@
+/*
```

Comment:
If we keep these classes, recommend nesting this class and the Wrapper class in a /yarn package, since they're specific to YARN.